cmd/vic-machine/configure/configure.go

```diff
@@ -184,6 +257,10 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 			v.Network.Nameservers = n.ExecutorConfig.Networks[k].Network.Nameservers
 		}
 	}
+	o.HostCertificate = n.HostCertificate
```

Comment:
do you need to check if the HostCertificate is set? you might not want to override existing value with a nil